Neos.Neos/Classes/Domain/Repository/UserRepository.php

```diff
@@ -34,21 +38,34 @@ public function findAllOrderedByUsername(): QueryResultInterface
             ->execute();
     }
 
+    /**
+     * @param bool $sortDescending
+     * @return QueryResultInterface
+     */
+    public function findAllOrderedByLastLoggedInDate(bool $sortDescending): QueryResultInterface
```

Comment:
I would rather add a generic method `findAllOrdered(string $field, string $direction)` that takes the field as parameter